pkg/httpclientutil/httpclientutil_windows.go

```diff
@@ -17,8 +19,7 @@ func NewHTTPClientWithSocketPath(socketPath string) (*http.Client, error) {
 	hc := &http.Client{
 		Transport: &http.Transport{
 			DialContext: func(ctx context.Context, _, _ string) (net.Conn, error) {
-				var d net.Dialer
-				return d.DialContext(ctx, "unix", socketPath)
+				return netutil.DialUnix("unix", socketPath)
```

Comment:
Same here, we don't want drop the context. We can work on this later if you think this context is not really used.